components/accept-financial-contributions/StripeOrBankAccountPicker.js

```diff
@@ -250,12 +238,26 @@ class StripeOrBankAccountPicker extends React.Component {
   }
 }
 
-const getBankAccountInfo = graphql(getCollectivePageQuery, {
+const hostQuery = gqlV2`
```

Comment:
Using GraphQL API v2, connectedAccounts is just being introduced.

I do remember I suggested re-using an existing query but I actually haven't found one that made sense. Sorry for the back and forth.